### ISSUE TITLE
All glyphs related to 'l', excluding itself, refer to l.ss02 (plain l) instead of standard 'l' (it has a bend).

### ISSUE THREAD

**Issue by cssobral2013** [Etcetera-Type-Co/Epilogue#8]
I'd like to ask if is this intentional.

**Comment by sursly** [comment]
Are you asking for it to refer to a new stylistic set, or reporting that you are getting a new stylistic set? 
The bend is intentional, and there is already a .ss01 "l" that is a straight stem.

**Comment by cssobral2013** [comment]
No. More specifically, the 'l' glyph on all diacrictics doesn't have a bend
while the 'l' without them has it.

Em qua, 3 de jun de 2020 11:49 AM, Tyler Finck <notifications@github.com>
escreveu:

**Comment by sursly** [comment]
Ah, thanks for specifying. I pushed an update last night that fixes this.
